index.html

```diff
@@ -14,26 +14,41 @@
   <body>
     <header class="header">
       <div class="container">
-        <div class="item-1"><img src="images/My Logo.png" /></div>
-        <div class="item-2"><img src="images/Hamburguer.png" /></div>
+        <a href=""><img src="images/MyLogo.png" alt="Logo" /></a>
+        <nav class="item-2"><img src="images/Hamburguer.png" alt="Menu" /></nav>
       </div>
     </header>
     <section>
-      <h1 class="first-title">I'm Omar Glad to see You</h1>
-      <p class="description">
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-    </section>
-    <section class="section-2">
-      <p class="connect">LET'S CONNECT</p>
-      <div class="container-2">
-        <img src="images/twiter.svg" alt="" />
-        <img src="images/Linkedin.svg" alt="" />
-        <img src="images/m.svg" alt="" />
-        <img src="images/github.svg" alt="" />
-        <img src="images/manito.svg" alt="" />
+      <div class="container-1">
```

Comment:
#### OPTIONAL 

I'll mark this as a suggestion rather than a requirement since the last reviewer missed it so you can implement it if you wish 🤓 

You could apply the background image to the section class rather than the `div` because as you see there is a white space on the sides. Be sure to get rid of any margin on the section to apply the background on all the screen 🤓
![Captura](https://user-images.githubusercontent.com/15807343/148423010-c967132a-981b-4a98-b8e1-0fa076b9fe0a.PNG)